.github/actions/build-and-publish-docker/action.yml

```
@@ -31,10 +31,14 @@ runs:
 
     - name: "Build docker image"
       shell: bash
-      run: ./scripts/docker-release/build_docker.sh "${{ inputs.release_version }}"
+      run: ./scripts/docker-release/build_docker.sh "$RELEASE_VERSION"
```

```suggestion
      run: ./scripts/docker-release/build_docker.sh "${RELEASE_VERSION}"
```

[minor] adding curly braces for extra clarity, doing the same for all other variables.